src/customers/controller/customers.controller.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  Body,
+  Controller,
+  Get,
+  NotFoundException,
+  Param,
+  Post,
+} from '@nestjs/common';
+import CustomerInDTO from '../dto/customer.indto';
+import CustomersMapper from '../mapper/customers.mapper';
+import { CustomersService } from '../service/customers.service';
+import CustomerOutDTO from '../dto/customer.outdto';
+import { ApiResponse } from '@nestjs/swagger';
+
+@Controller('customers')
+export class CustomersController {
+  constructor(
+    private customersMapper: CustomersMapper,
+    private customerService: CustomersService,
+  ) {}
+
+  @ApiResponse({ status: 200, description: 'The user was successfully found' })
+  @ApiResponse({ status: 404, description: "The user can't be found" })
+  @Get(':id')
+  async getCustomerById(
```

Comment:
Check what happens when you use an invalid UUID for this operation. 

e.g., `localhost:3000/customers/1`. We should never have an Internal server error. We need to handle bad input too.


Good job for returning a DTO to the client!